exploring_UV_extinction_curves/Exploring_UV_extinction_curves.ipynb

```diff
@@ -0,0 +1,1662 @@
+{
```

Comment:
<p>by visiting the URLs provided by <code>download_products</code>, you will be able ...</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/9/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>